python/lsst/ip/diffim/dipoleFitTask.py

```diff
@@ -102,10 +107,16 @@ def setDefaults(self):
                               "base_PixelFlags",
                               "base_SkyCoord",
                               "base_PsfFlux",
+                              "base_GaussianCentroid",
+                              #"base_GaussianFlux",
```

Comment:
base_GaussianFlux was commented out because it leads to a schema set-up failure:
```
Alias for 'slot_Shape_flag' must be defined before initializing 'base_GaussianFlux' plugin. {0}
lsst::pex::exceptions::LogicError: 'Alias for 'slot_Shape_flag' must be defined before initializing 'base_GaussianFlux' plugin.'
```
I'll remove this line now, but I've filed a ticket (DM-7458), as I would like to add base_GaussianFlux to the default list of plugins.